.github/workflows/ci.yml

```
@@ -53,6 +53,34 @@ jobs:
           artifacts_path: dist
           anaconda_nightly_upload_token: ${{ secrets.UPLOAD_TOKEN }}
 
+      - name: Build a wheel and a sdist
```

Too bad it's not possible to merge the `run` and `uses` steps into one, so maybe just change the names so it's not repeated, which may be helpful when quick looking the code
```suggestion
      - name: Build v0.0.2 wheel and sdist
```